packages/design-system/src/components/index.ts

```diff
@@ -13,6 +13,7 @@ export * from './Tag'
 export * from './Empty'
 export * from './Form'
 export * from './Popover'
+export * from './PopoverN'
```

Comment:
I think we can rename the old Popover to `PopoverLegacy`, and name the new popover as `Popover`

Reply:
Because changing the name to legacy would involve a lot of renaming, the choice was made to add the N suffix to the new version

The new version of popover will be developed for a while and will coexist with the old version

After that, it will use PopoverN for a period of time, then clear Popover and rename it Popver